test/reporters/min.spec.js

```diff
@@ -15,6 +17,12 @@ describe('Min reporter', function () {
     process.stdout.write = function (string) {
       stdout.push(string);
     };
+    useColors = Base.useColors;
```

Comment:
I'm going to assume this is necessary because it's easier to parse the output?

Reply:
Right, the other reporter tests do this